src/pages/shared/InputSelect/InputSelect.jsx

```diff
@@ -2,15 +2,15 @@ import React from "react";
 import PropTypes from "prop-types";
 import 'pages/shared/InputSelect/InputSelect.scss';
 
-function InputSelect({label, name, genres}) {
+function InputSelect({label, name, genres, value, handleMovieChange}) {
     return (
         <div className="input-select">
             <label className="input-select__label">{label}</label>
-                <select className="input-select__list" name={name}>
-                    {Object.entries(genres).map(([key, genre]) => (
-                        <option className="input-select__item" key={key}>{genre}</option>
-                    ))}
-                </select>
+            <select multiple={true} className="input-select__list" name={name} onChange={handleMovieChange} value={value}>
```

Comment:
I suppose it is enough just 'multiple' in our case...
 <select multiple className="input-select__list"

OR pass a prop to maintain both states 
function InputSelect({label, name, genres, value, handleMovieChange, multiple}) {
<select {multiple ? 'multiple' : ''}